meta2v2/meta2_utils.c

```diff
@@ -2445,8 +2447,15 @@ m2db_check_content_quality(struct m2v2_sorted_content_s *sorted_content,
 		}
 		return FALSE;
 	}
+
+	void _on_items(gpointer key UNUSED, struct oio_lb_selected_item_s *item, gpointer data UNUSED) {
+		g_free((char *) item->expected_slot);
+		g_free((char *) item->final_slot);
+	}
+
 	g_tree_foreach(sorted_content->metachunks,
 			(GTraverseFunc)_on_metachunk, NULL);
+	g_hash_table_foreach(chunk_items, (GHFunc)_on_items, NULL);
```

Comment:
We should define `oio_lb_selected_item_free` and pass that to the hash table constructor.